lnrpc/walletrpc/walletkit.proto

```diff
@@ -414,8 +414,8 @@ message ListAccountsResponse {
 }
 
 message ListAddressesRequest {
-    // An optional filter to only return accounts matching this name.
-    string name = 1;
+    // An optional filter to only return addresses matching this account.
+    string account = 1;
```

Comment:
Ah, I see you already renamed things in this commit. Never mind my earlier comments then.